test/runtests.jl

```diff
@@ -453,18 +453,16 @@ context("Matrix Math") do
 				fmm = inv(mfs)
 				@fact isapprox(fmm, mm)  --> true
 			end
-			if i <= 2
-			    context("expm(M)") do
-			        mm = expm(m)
-				    fmm = expm(mfs)
-				    @fact isapprox(fmm, mm)  --> true
-
-				    mm = expm(mc)
-				    fmm = expm(mfsc)
-				    @fact isapprox(fmm, mm)  --> true
-			    end
+			context("expm(M)") do
+				mm = expm(m)
+				fmm = expm(mfs)
+				@fact isapprox(fmm, mm)  --> true
+
+				mm = expm(mc)
+				fmm = expm(mfsc)
+				@fact isapprox(fmm, mm)  --> true
```

Comment:
(Please notice that between the trailing spaces removals there is an actual change to code here.)